src/main/java/com/mindee/MindeeClient.java

```diff
@@ -53,6 +55,54 @@ public DocumentToParse loadDocument(
     return new DocumentToParse(fileInBase64Code, filename);
   }
 
+  public <T extends Inference> PredictResponse<T> jobStatusForId(Class<T> type, String jobId) {
```

Comment:
The method should be named `parseQueued` for harmony with other SDKs.

Apologies, there was an error in the ticker description which made this quite ambiguous.